lib/rules/no-array-prototype-extensions.js

```diff
@@ -714,6 +714,16 @@ module.exports = {
           return;
         }
 
+        // ignore ember array initialization with A([]) or EmberArray([])
+        if (
+          node.type === 'CallExpression' &&
+          nodeInitializedTo &&
+          nodeInitializedTo.callee &&
+          (nodeInitializedTo.callee.name === 'A' || nodeInitializedTo.callee.name === 'EmberArray')
```

Comment:
We're going to have to check against the actual names these are imported under, if they're imported at all. See the `ImportDeclaration` visitor for example.

Reply:
Updated the logic to fetch the actual import name and also updated the test cases to include an import statement.